packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -99,6 +100,8 @@ export type Dispatcher = {
   useTransition(
     config: SuspenseConfig | void | null,
   ): [(() => void) => void, boolean],
+
+  unstable_avoidThisRender: typeof unstable_avoidThisRender,
```

Comment:
Let's write out the expected signature explicitly. We also don't need to make this name "unstable" prefixed since it's a private API. It just adds bytes.